script.js

```diff
@@ -1,4 +1,80 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
-};
+// variable assignments
+var scissors = "scissors";
+var paper = "paper";
+var stone = "stone";
+var reversedScissors = "reversed scissors";
+var reversedPaper = "reversed paper";
+var reversedStone = "reversed stone";
+var instructions =
+  'Type in "scissors" "paper" or "stone" to throw as your next hand. Good luck! <br>(Psst. You can add reversed before your hand to do inverted scissors papers stone!)';

```

Comment:
I like the additional instructions placed here that makes the game much easier to be played